src/app/containers/Ad/Amp/index.jsx

```diff
@@ -80,7 +80,11 @@ const AmpAd = ({ slotType }) => {
   const label = pathOr('Advertisement', ['advertisementLabel'], ads);
 
   return (
-    <div amp-access="toggles.ads.enabled" amp-access-hide="true">
+    <div
```

Comment:
Please change div to section as per https://github.com/bbc/simorgh/pull/7025/files#r450157934

Reply:
Done :)